bedrock/firefox/templates/firefox/features/independent.html

```diff
@@ -62,12 +62,21 @@ <h2 class="section-title"><span>{{ _('Features') }}</span></h2>
         <img src="{{ static('img/firefox/features/thumbnails/independent/mission.jpg') }}" alt="">
         <h3>{{ _('A browser on a mission') }}</h3>
         <p>
+        {% if l10n_has_tag('independent_mission_bug_1391287') %}
+        {% trans %}
+          Firefox is a browser with a conscience. As part of the technology non-profit
+          Mozilla, we fight for your online rights, keep corporate powers in check and
+          help educate developing countries work with allies around the world to develop
```

Comment:
I think this was probably a copy/paste error in the bug. This should probably be "and work with allies around the world to develop a healthier Internet", omitting the "help educate developing countries" part.